x/cash/dynamicfee.go

```diff
@@ -58,33 +58,41 @@ func NewDynamicFeeDecorator(auth x.Authenticator, ctrl Controller) DynamicFeeDec
 }
 
 // Check verifies and deducts fees before calling down the stack
-func (d DynamicFeeDecorator) Check(ctx weave.Context, store weave.KVStore, tx weave.Tx, next weave.Checker) (res weave.CheckResult, ferr error) {
+func (d DynamicFeeDecorator) Check(ctx weave.Context, store weave.KVStore, tx weave.Tx, next weave.Checker) (fres weave.CheckResult, ferr error) {
```

Comment:
`fres` is from Fee RESult right? Would `cres` (as in Check RESult`) a better name? Similarly `cerr`. :thinking: 

If yes, the same rule should apply to `Deliver` method - 'dres' and `derr`.

Reply:
I used the name for final result (as opposed to the local result). rres for "return result" seemed a bit like stutter.

But sure, I can use cres and dres.